README.md

```diff
@@ -58,6 +58,8 @@ how to embed it yourself.
 
 See `hcloud help` for a list of commands.
 
+You can also check out the [command reference](docs/reference) for help.
```

Comment:
Could we use `docs/references` plural please ?

Reply:
We can follow https://diataxis.fr/reference/ to the letter and use the singlar form, but in that case I'll need to update the fleeting plugin.